WebRoot/skins/_base/base4/skin.css

```diff
@@ -756,6 +756,28 @@ svg.ImgColumnDownArrow{
     fill: @IconColumnDownArrow@;
 }
 
+.ZmComposeView svg.ImgAttachClip {
+    width: 29px;
```

Comment:
should the svg.ImgAttachClip be having width:100%  to take it from its parent which in this would be div.ImgAttachClip ? 
Doesn't change things  but better to keep static css values at minimum places. 
Its fine if can't be done.